<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDetailsDialog</class>
 <widget class="QDialog" name="ConnectionDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>361</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="mailBoxList"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="destinationFolder"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Connection:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Server URL:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="serverURL"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="connectionName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="userName"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Destination Folder:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Mailbox(s):</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Ignore List:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="ignoreList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="updates">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Updates</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="all">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>238</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Poll Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="pollTime"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Retry Count:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="retryCount"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>connectionName</tabstop>
  <tabstop>serverURL</tabstop>
  <tabstop>userName</tabstop>
  <tabstop>password</tabstop>
  <tabstop>mailBoxList</tabstop>
  <tabstop>destinationFolder</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
